utils/swift_build_support/swift_build_support/shell.py

```diff
@@ -80,40 +88,74 @@ def call(command, stderr=None, env=None, dry_run=None):
             "': " + e.strerror)
 
 
+def capture(command, stderr=None, env=None, dry_run=None, print_command=True):
```

Comment:
I agree that False makes more sense as a default, but didn't want the methods to be inconsistent. There is also an argument to be made that we generally shouldn't usually be shelling out to query things, and if we are it might be just as useful for logging purposes to see what commands contributed to the build script behavior.

As the module stands I think keeping the methods consistent at the cost of slightly more verbose clients is the right tradeoff.